pages/treeherder.py

```diff
@@ -91,15 +125,27 @@ def unclassified_failure_count(self):
     def clear_filter(self):
         self.selenium.find_element(*self._clear_filter_locator).click()
 
+    def clear_filter_by_shortcut(self):
+        el = self.find_element(*self._result_sets_locator)
+        el.send_keys(Keys.CONTROL + Keys.SHIFT + 'f')
+
     def click_on_filters_panel(self):
         self.find_element(*self._filter_panel_locator).click()
 
     def click_on_active_watched_repo(self):
         self.find_element(*self._active_watched_repo_locator).click()
 
+    def click_on_in_progress_button(self):
```

Comment:
I was also thinking about the name, that it is misleading. This shortcut can do both show/unshow - on/off.
And also about the `method` - could you please give me an example?